tests/ops/op_math/test_adjoint_op.py

```diff
@@ -701,3 +685,18 @@ def circuit(x):
         expected_grad = np.cos(x)
 
         assert qml.math.allclose(grad, expected_grad)
+
+    def test_adj_batching(self):
+        """Test execution of the adjoint of an operation with batched parameters."""
+        dev = qml.device("default.qubit", wires=1)
+
+        @qml.qnode(dev)
+        def circuit(x):
+            Adjoint(qml.RX(x, wires=0))
+            return qml.expval(qml.PauliY(0))
+
+        x = qml.numpy.array([1.234, 2.34, 3.456])
+        res = circuit(x)
+
+        expected = np.sin(x)
+        assert qml.math.allclose(res, expected)
```

Comment:
Just for my understanding: Does this work because operations with `has_matrix=False` are decomposed and the decomposition of `Adjoint(qml.RX(x, wires=0))` into `qml.RX(-x, wires=0)` is broadcasting-compatible? :)

Reply:
Exactly.  Since it doesn't have a matrix, it isn't supported by the device.  Since it isn't supported, it gets decomposed.  The decomposition then supports broadcasting.

Reply:
Thanks!